programs-remote-listings/templates/shortcode-programs.php

```diff
@@ -41,6 +46,10 @@
                 <div class="rs-program-excerpt"><?php echo $RS_Connect->excerpt($program->text); ?></div>
             <?php endif; ?>
 
+            <?php if ($program->first_price && ! empty($show_first_price)) : ?>
+                <div class="rs-program-first-price">Starting at <?php echo RS_Currency::format_accounting($program->first_price); ?></div>
```

Comment:
@cloudEdwards it's clear you did not even test this. 
- RS_Currency::format_accounting() only exists in our main app. Also, the api is spitting it out formatted already. 
- the API is spitting out program->price_first not first_price.
- we need to tweak the fields to actually request that field.